AIOnboardingARMTemplates/existingClusterOnboarding.json

```diff
@@ -0,0 +1,89 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "aksResourceId": {
+            "type": "string",
+            "metadata": {
+                "description": "AKS Cluster Resource ID"
+            }
+        },
+        "aksResourceLocation": {
+            "type": "string",
+            "metadata": {
+              "description": "Location of the AKS resource e.g. \"East US\""
+            }
+        },
+        "openTelemetryLogsPort": {
+            "type": "int",
+            "metadata": {
+              "description": "Port to use for OpenTelemetry Logs"
+            },
+            "defaultValue": "28331"
+        },
+        "openTelemetryMetricsPort": {
+            "type": "int",
+            "metadata": {
+              "description": "Port to use for OpenTelemetry Metrics"
+            },
+            "defaultValue": "28333"
+        },
+        "resourceTagValues": {
```

Comment:
What is the usage for these tags?

Reply:
I think it is just used by customers to tag their deployments and resources. It is same as Azure tags: https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/tag-resources